### DISCUSSION TITLE
Unable to process gif

### DISCUSSION THREAD

**Discussion by mati-a** [libvips/libvips#3249] [Q&A] [answered]
Im not sure if it is a bug

## Bug report

**Describe the bug**
Failed to process a large gif when trying to copy or resize it.

**To Reproduce**
1. Download [this gif](https://giphy.com/gifs/i-love-you-deer-rudolph-zgNEQ7Q2tkRFmmdOXF)
2. Then run `vips copy path_to_gif.gif[n=-1] o.gif`
3. See this error:
```
vips cache : gifload filename="path_to_gif.gif" out=((VipsImage*) 0xaaaacc134020) n=72 flags=((VipsForeignFlags) VIPS_FOREIGN_SEQUENTIAL) access=((VipsAccess) VIPS_ACCESS_SEQUENTIAL) -
vips cache : copy in=((VipsImage*) 0xaaaacc134020) out=((VipsImage*) 0xaaaacc1341b0) -
vips cache+: linecache in=((VipsImage*) 0xaaaacc134ca0) out=((VipsImage*) 0xffff80016000) tile-height=16 access=((VipsAccess) VIPS_ACCESS_SEQUENTIAL) persistent=TRUE -
vips cache+: sequential in=((VipsImage*) 0xaaaacc134ca0) out=((VipsImage*) 0xaaaacc134e30) tile-height=16 -

(vips:29262): VIPS-WARNING **: 19:24:48.475: error in tile 0 x 15840
gifload: Image is defective, decoding aborted
```

**Expected behavior**
The gif is processed.

**Actual behavior**
An error is raised.

**Environment**
 - OS: Dockerized Debian GNU/Linux 11 (bullseye)
 - Vips: 8.10.5-2

**Additional context**
Im using vips with rails to create a file variant the use case is more complex but it is the same error. With `n=30` works but with higher values fails.

**Comment by jcupitt** [comment]
Hello, it's hard to tell what the file is from that link. I think giphy is sharing it as a webp.

Could you upload your gif to a file sharing site so I can be sure I have the same file?

**Comment by mati-a** [comment, marked as answer]
Hello, this is the [file](https://www.dropbox.com/s/4aum520qicu7djq/downloaded.gif?dl=0)

I just tested with the 8.13.3-2 version and it works. Thanks!
